common/src/main/java/com/wynntils/mc/mixin/PotionItemMixin.java

```diff
@@ -14,7 +14,7 @@
 
 @Mixin(PotionItem.class)
 public abstract class PotionItemMixin {
-    @Inject(method = "isFoil()B", at = @At("HEAD"), cancellable = true)
```

Comment:
Why remove the method signature?

Reply:
It is not needed, and actually, some intelij plugin even complained. 

As far as I know, by standard, method signatures and param types are left out when there is not any other method with the same name.

Reply:
I see now I messed it up completely. FIxing in separate PR.